ncp-web/js/ncp.js

```diff
@@ -25,8 +25,9 @@ window.onpopstate = function(event) {
     click_app($('#' + selectedID));
 };
 
-function errorMsg()
-{ 
+function errorMsg(e)
+{
+  console.log(e); 
```

Comment:
```suggestion
```

Reply:
Fixed in cd8451d318815945bae42ddda0a16b0a1ff1468a